src/test/java/AWS4Test.java

```diff
@@ -752,6 +789,7 @@ public void testMultipartCopyMultipleSizesLLAPIAWS4() {
 		try {
 			svc.putObject(new PutObjectRequest(src_bkt, key, file));
 		} catch (AmazonServiceException err) {
+                  // ALI NOTE: what's the point of this try statement
```

Comment:
I think this is left over from testing some issue putting objects, maybe a timeout when the file is too large or smth similar, but I don't see a good reason to have it there